data/FilterList.json

```diff
@@ -16589,5 +16590,18 @@
     "syntaxId": 8,
     "updatedDate": "2019-01-15T03:00:00",
     "viewUrl": "https://raw.githubusercontent.com/cbuijs/shallalist/master/webtv/urls"
-  }
-]
\ No newline at end of file
+  },
+  {
+    "id": 1453,
```

Comment:
Sorry, just merged #683 which used this id number. Nothing you did wrong, just a fault of the current process. Please bump this Id to the next unused number, then I'll merge. Thanks!